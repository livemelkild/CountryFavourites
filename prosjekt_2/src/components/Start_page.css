@media (prefers-reduced-motion: no-preference){
    #circle{
      animation: moveStart infinite 3s linear;
    }
  }


  @keyframes moveStart {
      0%{
        transform: rotate(1deg);
      }
      50%{
        transform: rotate(-1deg)
      }
      100%{
        transform: rotate(1deg);
      }
    }