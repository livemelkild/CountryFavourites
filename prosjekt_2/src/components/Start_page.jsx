import React from "react"
import {Link} from "react-router-dom";
import Style from "./Start_page.css";

export class Start_page extends React.Component{
    render() {
        return(
            <div id="circle">
            <Link className = "start_link" to="/country">
                <br/>
                <img src={process.env.PUBLIC_URL + "/Image/arrow.svg"}></img>
                <div className="start_side"> 
                    Utforsk landene du kan reise til
                </div>
            </Link>
            </div>
        );
    }
}

export default Start_page;