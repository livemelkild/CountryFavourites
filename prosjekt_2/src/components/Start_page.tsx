import React from "react"
import {Link} from "react-router-dom";
import "./Start_page.tsx";

export class Start_page extends React.Component{
    render() {
        return(
            <div className="content_box">
                <div id="circle">
                <Link className = "start_link" to="/country">
                    <br/>
                    <img src={process.env.PUBLIC_URL + "/Image/arrow.svg"}></img>
                    <div className="start_side"> 
                        Utforsk landene du kan reise til
                    </div>
                </Link>
                </div>
            </div>
        );
    }
}

export default Start_page;