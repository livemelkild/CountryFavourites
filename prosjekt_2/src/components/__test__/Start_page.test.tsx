import React from "react";
import renderer from "react-test-renderer";
import Start_page from "../Start_page";
import {BrowserRouter as Router, Switch, Route} from "react-router-dom";


it("renders without crashing", () => {
    const tree = renderer.create(<Router><Start_page /></Router>).toJSON();
    expect(tree).toMatchSnapshot();
});