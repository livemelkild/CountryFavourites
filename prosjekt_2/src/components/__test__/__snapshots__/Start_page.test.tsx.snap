// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="content_box"
>
  <div
    id="circle"
  >
    <a
      className="start_link"
      href="/country"
      onClick={[Function]}
    >
      <br />
      <img
        src="/Image/arrow.svg"
      />
      <div
        className="start_side"
      >
        Utforsk landene du kan reise til
      </div>
    </a>
  </div>
</div>
`;
